src/www/firewall_rules.php

```diff
@@ -477,6 +489,28 @@
                         $selected_if == 'FloatingRules'
                      )
                 ):
+				  if( $selected_if != $interface )
+                  {
+					$offset = $i;
+                    $interface = $selected_if;
+                  }
+                  $idx = $i-$offset;
+
+
+                  if ($filterent['category'] !== $category) {
+                    $category = $filterent['category']
+?>
+
+                  <tr class="category text-uppercase text-danger text-center font-weight-bold" data-category="<?=!empty($filterent['category']) ? $filterent['category'] : "";?>">
+                    <td colspan=2></td>
+                    <td colspan=3><?=$category?></td>
+                    <td></td>
+                    <td></td>
+                  </tr>
```

Comment:
Optically this doesn't look really nice on my end, I've attached a screenshot of what it looks like now
![image](https://user-images.githubusercontent.com/9783985/44310481-5e49ef80-a3d7-11e8-8974-8244a8644160.png)

Reply:
I think rendering issue is caused by an empty category for rule 3?